### PR TITLE
Update CHANGELOG for 2.18.0 release

### PR DESCRIPTION
**Pull request by JLXIA**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+# Release 2.18.0 16-09-2020
+
+- Add name prefix for backend/jwtProvider cluster (#330)
+- Split specifying root certs for backend vs sidestream SSL clients (#325)
+- Rich access logging of http request information (#316)
+- Support IP in backend address (#323)
+- For Cloud Run, report location with region instead of zone (#314)
```
Comment (nareddyt) [inline CHANGELOG.md:7]: Please add this one too: https://github.com/GoogleCloudPlatform/esp-v2/pull/318

Comment (JLXIA) [reply CHANGELOG.md:8]: Done.

**Review by nareddyt — APPROVED** [review]

**Review by qiwzhang — APPROVED** [review]

**Review by TAOXUY — APPROVED** [review]

**Comment by nareddyt** [comment]
@JLXIA you can ignore these test errors. It seems there was a merge conflict that our tests didn't catch when PR #330 and PR #322 were submitted. I have PR #332 to resolve them.

**Pull request merged**